CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -322,7 +325,8 @@ public let ATTRIBUTE_NODES: [Node] = [
     kind: .derivativeRegistrationAttributeArguments,
     base: .syntax,
     nameForDiagnostics: "attribute arguments",
-    documentation: "The arguments for the '@derivative(of:)' and '@transpose(of:)' attributes: the 'of:' label, the original declaration name, and an optional differentiability parameter list.",
+    documentation:
```

Comment:
Unfortunately, there isn’t. Really these doc comments should just be multi-line strings so that the generated documentation also has newlines to improve readability. But if we want to change that, I’d like to do it in a follow-up PR.